### PR TITLE
Updating Titanium Connection Request to test for Platform 4.0 links

### PR DESCRIPTION
**Pull request by jamilhassanspain**
- adding process end statement so process will be ended and not hang until it finished
- updating with links for the Platform 4.0

### CODE REVIEW

File: titanium-connection-test.js [hunk 1]
```diff
@@ -31,7 +31,10 @@ var async = require('async'),
 		'https://studio.appcelerator.com',
 		'https://www.appcelerator.com',
 		'https://api.appcelerator.net',
-		'https://registry.npmjs.org'
+		'https://registry.npmjs.org',
+		'https://appc-studio.appcelerator.com',
+		'https://software.appcelerator.com',
+		'https://web.appcelerator.com'
```
Comment (pinnamur) [inline titanium-connection-test.js]: Thanks @jamilspain-appc for adding those additional links. I think we still need to ``platform.appcelerator.com`` to the list.

Also, we can consider to remove ``studio.appcelerator.com, my.appcelerator.com, api.appcelerator.net`` as these servers are going away very soon.

**Comment by ingo** [comment]
I would keep all the servers for now. @jamilhassanspain can you add platform.appcelerator.com?

**Comment by jamilhassanspain** [comment]
ok, I'll make these updates.

**Comment by ingo** [comment]
Approved. Merging.

**Pull request merged**